banzai/tests/utils.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, image_multiplier=1.0, nx=101, ny=103, name='test_image', read
         if meta is not None:
             self.meta = meta
         else:
-            self.meta = Header()
+            self.meta = Header({'SATURATE': 1, 'GAIN': 1, 'MAXLIN': 1})
```

Comment:
Minor nitpick. If we're not using these header keywords in many other tests, we may just want to specify them when instantiating the `FakeCCDData` object in the test itself, rather than having them automatically hardcoded/populated in the constructor.

Reply:
Also a saturate keyword of 1 and maxlin of 1 has the potential to break things.